react-cardknox-ifields/README.md

```diff
@@ -394,29 +394,17 @@ export default class App extends React.Component {
         <td>Turn 3DSecure on and off</td>
     </tr>
     <tr>
-        <td>waitForResponse</td>
-        <td>Boolean</td>
-        <td>Determine whether iFields should wait for a response from 3DSecure before getting the token</td>
-    </tr>
-    <tr>
-        <td>waitForResponseTimeout</td>
-        <td>Number</td>
-        <td>The 3DSecure response timeout in milli-seconds. The default value is 2000 (2 seconds).</td>
-    </tr>
-    <tr>
-        <td>amount</td>
-        <td>Number</td>
-        <td>The transaction amount</td>
-    </tr>
-    <tr>
-        <td>month</td>
-        <td>Number</td>
-        <td>The 2-digit card expiration month</td>
+        <td>environment</td>
+        <td>String</td>
+        <td>`Staging` or `Production`
+        <br/>
+        These values can be imported from the package
+        </td>
     </tr>
     <tr>
-        <td>year</td>
-        <td>Number</td>
-        <td>The 2-digit card expiration year</td>
+        <td>handle3DSResults</td>
+        <td>Function</td>
+        <td>A callback to handle the 3DS results</td>
```

Comment:
Will developers know how to use this callback?
This is the ifields public documentation https://docs.cardknox.com/cardknox-products/ifields#ifields-with-3d-secure-authentication

Reply:
I'll link the doc, thanks